### ISSUE TITLE
:en is not a valid locale (I18n::InvalidLocale)

### ISSUE THREAD

**Issue by vilde1** [leapcode/amber#2]
Hi,

I'm running amber-0.3.2 in debian testing. I installed it via the suggested way,  that is 'gem install amber'.

When running 'amber server' inside my "riseup_help" directory I get this error:

```
amber server      
/var/lib/gems/2.1.0/gems/i18n-0.7.0.beta1/lib/i18n.rb:284:in `enforce_available_locales!': :en is not a valid locale (I18n::InvalidLocale)
	from /var/lib/gems/2.1.0/gems/i18n-0.7.0.beta1/lib/i18n/config.rb:34:in `default_locale='
	from /var/lib/gems/2.1.0/gems/i18n-0.7.0.beta1/lib/i18n.rb:35:in `default_locale='
	from /var/lib/gems/2.1.0/gems/amber-0.3.2/lib/amber/site_configuration.rb:96:in `cleanup'
	from /var/lib/gems/2.1.0/gems/amber-0.3.2/lib/amber/site_configuration.rb:79:in `initialize'
	from /var/lib/gems/2.1.0/gems/amber-0.3.2/lib/amber/site_configuration.rb:46:in `new'
	from /var/lib/gems/2.1.0/gems/amber-0.3.2/lib/amber/site_configuration.rb:46:in `load'
	from /var/lib/gems/2.1.0/gems/amber-0.3.2/lib/amber/site.rb:17:in `initialize'
	from /var/lib/gems/2.1.0/gems/amber-0.3.2/lib/amber/cli.rb:66:in `new'
	from /var/lib/gems/2.1.0/gems/amber-0.3.2/lib/amber/cli.rb:66:in `server'
	from /var/lib/gems/2.1.0/gems/amber-0.3.2/bin/amber:79:in `main'
	from /var/lib/gems/2.1.0/gems/amber-0.3.2/bin/amber:82:in `<top (required)>'
	from /usr/local/bin/amber:23:in `load'
	from /usr/local/bin/amber:23:in `<main>'
```

**Comment by elijh** [comment]
resolved by 6ab62a44a605c594c93cafe67f0387d60320dce9

**Comment by elijh** [comment]
install v. 0.3.4 for the fix.
